pypesto/visualize/profile_cis.py

```diff
@@ -85,3 +87,105 @@ def profile_cis(
         ax.plot(ub, parameters_ind, "k--", marker="+")
 
     return ax
+
+
+def profile_nested_cis(
+        result: Result,
+        confidence_levels: Sequence[float] = (0.95, 0.9),
+        profile_indices: Sequence[int] = None,
+        profile_list: int = 0,
+        colors: Sequence = None,
+        ax: matplotlib.axes.Axes = None,
+        rotation: Literal["v", "h"] = "v",
+):
+    """
+    Plot approximate nested confidence intervals based on profiles.
+
+    Parameters
+    ----------
+    result:
+        The result object with profiling results.
+    confidence_levels:
+        The confidence levels in (0,1), which are translated to an approximate
+        threshold assuming a chi2 distribution, using
+        `pypesto.profile.chi2_quantile_to_ratio`.
+    profile_indices:
+        List of integer values specifying which profiles should be plotted.
+        Defaults to the indices for which profiles were generated in profile
+        list `profile_list`.
+    profile_list:
+        Index of the profile list to be used.
+    colors:
+         A color for each confidence interval.
+    ax:
+        Axes object to use. Default: Create a new one.
+    rotation:
+        Rotation of the plot, either vertical or horizontal.
+    """
+    # extract problem
+    problem = result.problem
+    # extract profile list
+    profile_list = result.profile_result.list[profile_list]
+
+    if colors is None:
+        colors = [None]*len(confidence_levels)
```

Comment:
I would vote for a default differing color, either making the blue brighter with each confidence level or increasing the alpha value.